pkg/plugins/manager/loader/loader.go

```diff
@@ -164,7 +164,7 @@ func (l *Loader) loadPlugins(ctx context.Context, src plugins.PluginSource, foun
 	for _, p := range verifiedPlugins {
 		err := l.pluginInitializer.Initialize(ctx, p)
 		if err != nil {
-			return nil, err
+			l.log.Error("Could not initialize plugin", "pluginId", p.ID, "err", err)
```

Comment:
I think we should skip the remainder of the process for those that cannot initialize, because if not they will still be registered and expected to function. We may want a new slice here (`initializedPlugins` for example) that can take only plugins from `verifiedPlugins` where there is no err from calling `Initialize`